test/test_functions

```diff
@@ -509,6 +509,10 @@ cvmfs_mount() {
     sudo sh -c "echo \"CVMFS_DEBUGLOG=$CVMFS_TEST_DEBUGLOG\" >> /etc/cvmfs/default.local" || return 100
   fi
 
+  if [ "x$CVMFS_TEST_QUOTED_PARAM_KEY_VAL" != "x" ]; then
+    sudo sh -c "echo '${CVMFS_TEST_QUOTED_PARAM_KEY_VAL}' >> /etc/cvmfs/default.local" || return 100
+  fi
```

Comment:
Is this needed? You can pass multiple `key=value` parameters to `cvmfs_mount`, like `cvmfs_mount lhcb.cern.ch a=b c=d`

Reply:
im open to suggestions but for me this was  the only way how i could get 
`CVMFS_HTTP_TRACING_HEADERS="h1:test|CVMFS-X-h2:ff|X-CVMFS-h3:12_ad |  h4  : 12fs_? "` correctly into the config. with the white spaces and the quotes around.

Reply:
I see. That's indeed a special case. Ok, let's keep it as is then.